homeassistant/components/aladdin_connect/const.py

```diff
@@ -1,22 +1,18 @@
-"""Platform for the Aladdin Connect cover component."""
-
-from __future__ import annotations
+"""Constants for the Aladdin Connect Genie integration."""
 
 from typing import Final
 
 from homeassistant.components.cover import CoverEntityFeature
-from homeassistant.const import STATE_CLOSED, STATE_CLOSING, STATE_OPEN, STATE_OPENING
 
-NOTIFICATION_ID: Final = "aladdin_notification"
-NOTIFICATION_TITLE: Final = "Aladdin Connect Cover Setup"
+DOMAIN = "aladdin_connect"
+CONFIG_FLOW_VERSION = 2
+CONFIG_FLOW_MINOR_VERSION = 1
 
-STATES_MAP: Final[dict[str, str]] = {
-    "open": STATE_OPEN,
-    "opening": STATE_OPENING,
-    "closed": STATE_CLOSED,
-    "closing": STATE_CLOSING,
-}
+OAUTH2_AUTHORIZE = "https://app.aladdinconnect.com/login.html"
+OAUTH2_TOKEN = "https://twdvzuefzh.execute-api.us-east-2.amazonaws.com/v1/oauth2/token"
 
-DOMAIN = "aladdin_connect"
+API_URL = "https://twdvzuefzh.execute-api.us-east-2.amazonaws.com/v1"
+API_KEY = "k6QaiQmcTm2zfaNns5L1Z8duBtJmhDOW8JawlCC3"
 SUPPORTED_FEATURES: Final = CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE
-CLIENT_ID = "1000"
+
+SCAN_INTERVAL = 20
```

Comment:
Not used

Reply:
How do I set the scan interval for polling? The documentation says this:

> You can control the polling interval for your integration by defining a SCAN_INTERVAL constant in your platform. Careful with setting this too low. It will take up resources in Home Assistant, can overwhelm the device hosting the API or can get you blocked from cloud APIs. The minimum allowed value is 5 seconds.

```
from datetime import timedelta

SCAN_INTERVAL = timedelta(seconds=5)
```
Where should I define this constant?

Reply:
you need to define it inside `cover.py`

Reply:
The default scan interval for covers is every 15 seconds, which is what is being used now.